### PR TITLE
Akka HTTP 10.1.8 release notes

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: blog/_posts/2019-03-21-akka-http-10.1.8-released.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+title: Akka HTTP 10.1.8 Released
+author: Arnout
+short: We are happy to announce the 10.1.8 release of Akka HTTP
+category: news
+tags: [releases]
+---
+
+Dear hakkers,
+
+We are happy to announce the 10.1.8 release of Akka HTTP. 10.1.7 is the ninth release in the 10.1.x series of Akka HTTP.
```
Comment (jlprat) [inline blog/_posts/2019-03-21-akka-http-10.1.8-released.md]: Typo. 1.1.8

File: blog/_posts/2019-03-21-akka-http-10.1.8-released.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+### Changes since 10.1.7
+
+For a full overview you can also see the [10.1.8 milestone](https://github.com/akka/akka-http/milestone/47?closed=1):
+
+#### Improvements
+
+##### akka-http-core
+
+* Do not include error details in HTTP response [#2314](https://github.com/akka/akka-http/pull/2314)
+* More reliable detection of HTTPS => HTTP connection attempts [#1886](https://github.com/akka/akka-http/pull/2314)
```
Comment (johanandren) [inline blog/_posts/2019-03-21-akka-http-10.1.8-released.md]: Wrong link url, the next one as well

File: blog/_posts/2019-03-21-akka-http-10.1.8-released.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+#### Bug fixes
+
+* Response shouldn't override connection header when connection is force-closed [#2403](https://github.com/akka/akka-http/issues/2403)
+* Disable to append charset=UTF-8 by default in `Content-Type: application/x-www-form-urlencoded` FormData [#91](https://github.com/akka/akka-http/issues/91)
+* Fix int overflow in sliceBytesTransformer [#2440](https://github.com/akka/akka-http/issues/2440)
+* LogRequest not respecting LogLevel in Java [#2418](https://github.com/akka/akka-http/issues/2418)
+
+
+For this release we had the help of 24 contributors – thank you all very much!
```
Comment (jlprat) [inline blog/_posts/2019-03-21-akka-http-10.1.8-released.md]: Now I see this, 24 contributors is wrong, I'm afraid. I can only count 10 in the table below

Comment (raboof) [reply blog/_posts/2019-03-21-akka-http-10.1.8-released.md]: Oops! Actually it seems 24 was correct but somehow the list got messed up, not sure what happened there. Updating.

**Pull request merged**